lib/oauthdb/utils.js

```diff
@@ -20,15 +20,32 @@ module.exports = {
     if (err instanceof error) {
       return err;
     }
+    if (! err.errno) {
+      throw err;
```

Comment:
What response is sent to the user if this occurs?

Reply:
It will bubble up and be converted into a `999 Unexpected Error`; I'll add a comment to this effect.